Mathlib/GroupTheory/GroupAction/SubMulAction.lean

```diff
@@ -161,41 +161,57 @@ structure SubMulAction (R : Type u) (M : Type v) [SMul R M] : Type v where
   /-- The carrier set is closed under scalar multiplication. -/
   smul_mem' : ∀ (c : R) {x : M}, x ∈ carrier → c • x ∈ carrier
 
+/-- A SubAddAction is a set which is closed under scalar multiplication. -/
+structure SubAddAction (R : Type u) (M : Type v) [VAdd R M] : Type v where
+  /-- The underlying set of a `SubAddAction`. -/
+  carrier : Set M
+  /-- The carrier set is closed under scalar multiplication. -/
+  vadd_mem' : ∀ (c : R) {x : M}, x ∈ carrier → c +ᵥ x ∈ carrier
+
+attribute [to_additive] SubMulAction
```

Comment:
Can you swap `SubAddAction` and `SubMulAction` around and tag `SubMulAction` with `to_additive` straight away?